From 27dc2349fec6087c0c8cbbf9151b3c37964c28e5 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?St=C3=A9phane=20Marchesin?= <marcheu@chromium.org>
Date: Wed, 20 Feb 2013 15:37:36 -0800
Subject: [PATCH] Disable bo recycling in libva

On snb and ivb, if you use the same command buffer for video commands
and then for 3d commands just after that, the GPU will lock up. This
seems to be a hardware bug which I haven't figured out yet.

This disables buffer recycling in libva to work around the issue.
---
 src/intel_memman.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/intel_memman.c b/src/intel_memman.c
index 7d56e96..7dd64db 100644
--- a/src/intel_memman.c
+++ b/src/intel_memman.c
@@ -36,7 +36,6 @@ intel_memman_init(struct intel_driver_data *intel)
 {
     intel->bufmgr = intel_bufmgr_gem_init(intel->fd, BATCH_SIZE);
     assert(intel->bufmgr);
-    intel_bufmgr_gem_enable_reuse(intel->bufmgr);
 
     return True;
 }
-- 
1.8.1.3

